pkg/resources/resource_monitor.go

```diff
@@ -183,9 +183,9 @@ func ReadResourceMonitor(data *schema.ResourceData, meta interface{}) error {
 
 	// Credit quota is the only integer
```

Comment:
not an integer, maybe worth fixing that comment

Reply:
done